tests/_support/Log/Handlers/TestHandler.php

```diff
@@ -47,8 +47,8 @@ public function __construct(array $config)
      * will stop. Any handlers that have not run, yet, will not
      * be run.
      *
-     * @param $level
-     * @param $message
+     * @param string $level
+     * @param string $message
```

Comment:
Was this change made by dependabot?

Reply:
No. php-cs-fixer will remove the docblocks through `no_superfluous_phpdoc`. but on a re-run, it will add `@param mixed $level` through `phpdoc_add_missing_param_annotation`

Reply:
So I added the correct docblocks

Reply:
I think you should add another commit because the commit is for *Updates the requirements on friendsofphp/php-cs-fixer*.

Reply:
Sure. I messed up earlier (don't know what happened then). I'll break the commit.

Reply:
Thanks!